--- baghira-0.3s/widget-engine/qeffects.cpp.orig	2004-06-13 10:52:53.000000000 +0200
+++ baghira-0.3s/widget-engine/qeffects.cpp	2004-06-13 20:54:56.447964872 +0200
@@ -36,11 +36,7 @@
 **********************************************************************/
 
 //#include <qapplication.h>
-#ifndef QT_EFF
-#define QT_EFF
 #include "baghira.h"
-#undef QT_EFF
-#endif // QT_H
 #ifndef QT_H
 #include <qnamespace.h>
 #endif // QT_H
